jenkins.yaml

```diff
@@ -1,13 +1,21 @@
 package:
   name: jenkins
   version: "2.448"
-  epoch: 0
-  description:
+  epoch: 1
+  description: Open-source CI/CD application.
   copyright:
     - license: MIT
   dependencies:
     runtime:
+      # Jenkins requires both bash and busybox.
+      - bash
+      - busybox
+      - coreutils
+      - glibc-locale-en
+      - openjdk-17-default-jvm
+      - openjdk-17-jre
```

Comment:
This prevents us from running in higher JRE environments

Reply:
I would wait for @stormqueen1990 and @rawlingsj on this

Reply:
Jenkins supports multiple Java versions - I can leave these specific settings in as image runtime dependencies, but there might be tweaks needed to the melange for the particular java version and they'd essentially be different version streams?

Reply:
Jenkins supports multiple Java versions at any given time. I can leave these specific settings in as image runtime dependencies, but there might be tweaks needed to the melange for the particular java version builds, and they'd essentially be different version streams.

Also passing them in at the image level would still mean it'd be compiled with a different JDK version. i.e at the minute JDK17, and run with whatever is passed in at the image level.

With that in mind it seems cleaner to define them in here and split to separate melange if we need to support concurrent JDK version builds / runtimes per release - was my thinking here

Reply:
Jenkins is compiled with Java 11 and run with different JRE Versions 11,17,21, please see https://hub.docker.com/r/jenkins/jenkins/tags

Reply:
Splitting of melange files is not needed here , instead compelling with JDK11 would be good instead

Reply:
Thanks @ajayk - in that case i'll flip over to jdk11 for buildtime and drop the runtimes from melange

Reply:
Applied the suggested change